gym_sts/autoencoder/train.py

```diff
@@ -234,15 +262,27 @@ def train():
                 results=results, prefix=f"Batch: {batch_num+1}/{total_batches}"
             )
 
+            to_log = dict(
+                loss=results["loss"].double(),
+                top1=results["top1"].double(),
+                epoch=epoch + batch_num / total_batches,
+            )
+            wandb.log(dict(train=to_log), step=train_step)
+
+            train_step += 1
+
         val_results = get_loss(tree.flatten(valid_tensor))
         val_loss = val_results["loss"]
         print_results(val_results, "Validation:")
         val_losses.append(float(val_loss))
         scheduler.step(val_loss)
 
-    for epoch in range(10):
-        print(f"Epoch {epoch}")
-        train()
+        to_log = dict(
+            loss=val_results["loss"].double(),
+            top1=val_results["top1"].double(),
+            epoch=epoch + 1,
+        )
+        wandb.log(dict(valid=to_log), step=train_step)
```

Comment:
Could we be more specific, `validation=`?

Reply:
Done